backends/p4tools/common/lib/format_int.cpp

```diff
@@ -16,6 +17,12 @@
 
 namespace P4Tools {
 
+static constexpr uint8_t IPV4_BYTE_SIZE = 4U;
+static constexpr uint8_t IPV6_BYTE_SIZE = 16U;
+static constexpr uint8_t MAC_BYTE_SIZE = 6U;
+/// Chunk size is 8 bits, i.e., a byte.
+static constexpr uint8_t CHUNK_SIZE = 8U;
```

Comment:
Would it be better to inline this number?
We are using uint8_t in several places, and several functions use "Bytes" in function and variable names. So it's not like this is a parameter that can be changed? I guess I don't see the motivation for declaring a global variable for this?